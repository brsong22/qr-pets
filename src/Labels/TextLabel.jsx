import React from 'react';
import PropTypes from 'prop-types';

const TextLabel = ({ label, value }) => (
  <div>
    {label}
    :&nbsp;
    {value}
  </div>
);

TextLabel.propTypes = {
  label: PropTypes.string,
  value: PropTypes.string,
};

TextLabel.defaultProps = {
  label: '',
  value: '',
};

export default TextLabel;
