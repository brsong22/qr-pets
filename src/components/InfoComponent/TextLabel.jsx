import React from 'react';
import PropTypes from 'prop-types';

const TextLabel = ({ label, value }) => (
  <div>
    {`${label} ${value}`}
  </div>
);

TextLabel.propTypes = {
  label: PropTypes.string,
  value: PropTypes.string,
};

TextLabel.defaultProps = {
  label: '',
  value: '',
};

export default TextLabel;
