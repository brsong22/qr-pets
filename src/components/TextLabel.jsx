import React from 'react';
import PropTypes from 'prop-types';

const TextLabel = ({ label, value }) => (
  <div>
    {`${label} ${value}`}
  </div>
);

TextLabel.propTypes = {
  label: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
};

TextLabel.defaultProps = {
  label: '',
  value: '',
};

export default TextLabel;
