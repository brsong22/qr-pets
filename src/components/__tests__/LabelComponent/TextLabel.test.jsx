
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TextLabel from '../../LabelComponent/TextLabel';

describe('TextLabel', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<TextLabel />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
