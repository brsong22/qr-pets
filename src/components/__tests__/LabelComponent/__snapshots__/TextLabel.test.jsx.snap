// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextLabel renders as expected 1`] = `
<div>
  :Â 
</div>
`;
